hawc/main/celery.py

```diff
@@ -28,4 +29,8 @@ def debug_task(self):
         "task": "hawc.apps.lit.tasks.update_pubmed_content",
         "schedule": 3600,
     },
+    "assessment-delete_old_jobs-1-week": {
+        "task": "hawc.apps.assessment.tasks.delete_old_jobs",
+        "schedule": timedelta(weeks=1),
```

Comment:
what do you prefer, time in seconds or the use of `timedelta`? Just so I can make it uniform

Reply:
oh i like timedelta. Can we do this daily instead? And perhaps the pubmed content can be daily too; hourly seems too frequent